DUI3-DX/DUI3/Speckle.Connectors.DUI/Bindings/ISendBinding.cs

```diff
@@ -27,6 +28,9 @@ public interface ISendBinding : IBinding
 public class SenderModelCard : ModelCard
 {
   public ISendFilter SendFilter { get; set; }
+
+  [JsonIgnore]
+  public HashSet<string> ChangedObjectIds { get; set; } = new();
```

Comment:
What was this needed for?

Reply:
This is needed for change detection on send, and it has been added a few weeks back to dui3/dev, but didn't make its way to dx.